build.rs

```diff
@@ -4,19 +4,38 @@ use gl_generator::{Registry, Api, Profile, Fallbacks};
 use std::env;
 use std::fs::File;
 use std::path::Path;
+use std::collections::BTreeMap;
 
 fn main() {
     let out_dir = env::var("OUT_DIR").unwrap();
 
     let dest = Path::new(&out_dir);
 
     let mut file = File::create(&dest.join("gl_bindings.rs")).unwrap();
+    let mut alliases = BTreeMap::new();
 
-    Registry::new(Api::Gl, (4, 5), Profile::Core, Fallbacks::All, [
-        "GL_EXT_texture_filter_anisotropic",
-        "GL_ARB_draw_buffers_blend",
-        "GL_ARB_program_interface_query",
-        ])
+    alliases.insert("BindVertexArray", vec!["BindVertexArrayARB"]);
```

Comment:
These 4 lines don't do shit, but I'm too lazy to remove them, as they are harmless.